examples/azure/build-and-deploy-multiple-artifacts.yml

```
@@ -0,0 +1,188 @@
+#
+# The example shows how to build, upload and deploy Mender Artifacts for two different devices.
+#
+# 'MENDER_SERVER_ACCESS_TOKEN' secret is required to be set in Azure DevOps pipeline settings.
+#
+
+trigger:
+- main
+
+
+variables:
+- name: MENDER_SERVER_URL # Mender server URL
+  value: https://hosted.mender.io
+- name: MENDER_ARTIFACT_NAME_ONE # First Mender Artifact file name
+  value: artifact_one.mender
+- name: MENDER_ARTIFACT_NAME_TWO # Second Mender Artifact file name in the `MENDER_ARTIFACTS_FOLDER` folder
+  value: artifact_two.mender
+- name: MENDER_RELEASE_NAME_ONE # The name of Mender release for Mender Artifact name `MENDER_ARTIFACT_NAME_ONE`
+  value: release-one-1.0
+- name: MENDER_RELEASE_NAME_TWO # The name of Mender release for Mender Artifact name `MENDER_ARTIFACT_NAME_TWO`
+  value: release-two-1.0
+- name: MENDER_DEVICES_GROUP_NAME_ONE # The name of Mender devices group a deployment with Mender Artifact name `MENDER_ARTIFACT_NAME_ONE` will be triggered to
+  value: group-one
+- name: MENDER_DEVICES_GROUP_NAME_TWO # The name of Mender devices group a deployment with Mender Artifact name `MENDER_ARTIFACT_NAME_TWO` will be triggered to
+  value: group-two
+
+
+resources:
+  # Use other repositories for templates
+  # https://learn.microsoft.com/en-us/azure/devops/pipelines/process/templates
+  repositories:
+    - repository: templates
+      type: github
+      name: mendersoftware/mender-ci-workflows
+      ref: refs/heads/master
+      # GitHub service connection
+      # https://learn.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints
+      endpoint: github_service_connection
+
+
+jobs:
+#
+# Builds Mender Artifacts using 'mender-artifact' tool
+#
+- job: build_mender_artifact_one
+  pool:
+    vmImage: ubuntu-latest
+  # conrainer with pre-installed mender-artifact and mender-cli tools
+  container: mendersoftware/mender-ci-tools:mender-master
+  steps:
+  # Creates Mender Artifact
```

I suggest using imperative wording in the headers:

```suggestion
  # Create a Mender Artifact
```